internal/fs/fs.go

```diff
@@ -12,117 +12,103 @@ import (
 
 type CloudStashFs struct {
 	manager *manager.Manager
-
```

Comment:
I think this empty line should stay, it separates the struct's own parameters and "inherited" parameters.

Reply:
@paddlesteamer Makes sense, but I think a comment would be helpful indicating the inherited fields and how are they being used. I found it hard to understand what it stood for at the first glance and didn't realize that it was in fact an inherited field.

Reply:
@paddlesteamer For instance, I don't see how `fuse.DefaultFileSystem` is being used, it doesn't seem to be referenced  from our code.

Reply:
Comments are nice of course, no argument against it but in `struct`s if it doesn't have a type it is inherited :D

Reply:
> @paddlesteamer For instance, I don't see how `fuse.DefaultFileSystem` is being used, it doesn't seem to be referenced from our code.

Ok, now I understand, I received a lot of questions about fuse implementation from you (about how nothing is referenced from our code), so let's take you [here](https://github.com/paddlesteamer/go-fuse-c/blob/master/example/hello/hello.go) and it's c [equivalent](https://www.maastaar.net/fuse/linux/filesystem/c/2016/05/21/writing-a-simple-filesystem-using-fuse/) to understand how fuse works. These are the tutorials I learned from.